### PR TITLE
added example for internal ELB to elb_v2.

### PR DESCRIPTION
**Pull request by reiki4040**
Hello, I found internal ELB configuration from source code and PR(https://github.com/eagletmt/hako/pull/5).
I could not find it first time, so I added to example for easy to find.

### CODE REVIEW

File: examples/hello-lb-v2.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ scheduler:
   elb_v2:
     # VPC id where the target group is located
     vpc_id: vpc-WWWWWWWW
+    # If you want internal ELB, then use 'scheme'. (ex. internal service that like microservice inside VPC)
+  # scheme: internal
```
Comment (eagletmt) [inline examples/hello-lb-v2.yml]: ill-indented?

Comment (reiki4040) [reply examples/hello-lb-v2.yml]: I had thought that same indent to vpc_id.
however, it looks like misreadable indent. I did fixed it.

**Comment by eagletmt** [comment]
thanks!

**Comment by reiki4040** [comment]
thanks!

**Pull request merged**